mibs/siae/IANA-ADDRESS-FAMILY-NUMBERS.mib

```diff
@@ -0,0 +1,130 @@
+
+  IANA-ADDRESS-FAMILY-NUMBERS-MIB DEFINITIONS ::= BEGIN
```

Comment:
This is an example of a standard mib.  You can find it here: mibs/IANA-ADDRESS-FAMILY-NUMBERS-MIB.  Delete it and any others there are copies of in the mibs folder.

Also, rename the rest of the mibs with the script `./scripts/rename-mibs.php mibs/siae/`

Reply:
Will do.